lib/streamlit/connections/sql_connection.py

```diff
@@ -171,14 +169,8 @@ def _query(
             **kwargs,
         )
 
-    @contextmanager
-    def session(self) -> Iterator["Session"]:
-        """A thin wrapper around SQLAlchemy Session context management.
-
-        This allows us to write
-            `with conn.session() as session:`
-        instead of importing the sqlalchemy.orm.Session object and writing
-            `with Session(conn._instance) as session:`
+    def session(self) -> "Session":
```

Comment:
Should this be a `@property` as well?

Reply:
Ah, yeah. I think it should be for consistency. Heads up @sfc-gh-jcarroll

Reply:
👍